cityofaustin/atd-moped
# Create more component types

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/10882

I believe it'd be in our best interest to wait to merge this to main until after we run the [transpose geometry scripts](https://github.com/cityofaustin/atd-moped/pull/792/files#diff-e1f48ee7a96cf574e0c4192719d0afe3cf9efe39771097d2f06bd2dd5f9c8953) for the last time.


## Testing
**URL to test:** Local

**Steps to test:**
1. Start Moped; visit the data dictionary: [`/moped/dev/lookups`](https://localhost:3000/moped/dev/lookups)
2. Confirm the component types listed in [the issue](https://github.com/cityofaustin/atd-data-tech/issues/10882) are present in the data dictionary.
3. Add a few of these new components to a project's map

---
#### Ship list
